src/AppSettings.cpp

```diff
@@ -305,6 +305,12 @@ void AppSettings::setDeviceName(const QString& address, const QString& deviceNam
     emit deviceNamesChanged(deviceNames());
 }
 
+void AppSettings::resetDeviceNames()
```

Comment:
Going by your other PR #84 it seems like this not being exposed through remoteobjects from AppSettings is a leftover from the initial thoughts during implementation. Probably want to do it here, rather than BTConnectionManager. Hope that isn't too much of an assumption, just seems like the sort of thing that might be happening :)

Reply:
I added comments like that:

```
/// We have access to this method only from the Service
/// See BTConnectionManager::clearDeviceNames() method
void clearDeviceNames();
```

I leave that method at the `AppSettings` class and do not call it from the QML because we use `setDeviceName()` method from `BTConnectionManager` class, so it would be correct if we call the `clearDeviceNames()` method from the `BTConnectionManager` class too.

Reply:
Right, that's out of scope of this patch, you're right :) Probably want to refactor that at some point, but for now, right choice :)